langchain4j-bedrock/src/test/java/dev/langchain4j/model/bedrock/BedrockChatModelIT.java

```diff
@@ -1,28 +1,137 @@
 package dev.langchain4j.model.bedrock;
 
+import static dev.langchain4j.internal.Utils.readBytes;
+import static org.assertj.core.api.Assertions.assertThat;
+
 import dev.langchain4j.data.message.AiMessage;
+import dev.langchain4j.data.message.ChatMessage;
+import dev.langchain4j.data.message.ImageContent;
 import dev.langchain4j.data.message.UserMessage;
+import dev.langchain4j.model.bedrock.BedrockMistralAIChatModel.Types;
 import dev.langchain4j.model.output.FinishReason;
 import dev.langchain4j.model.output.Response;
 import dev.langchain4j.model.output.TokenUsage;
+import java.util.Arrays;
+import java.util.Base64;
+import java.util.List;
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import software.amazon.awssdk.regions.Region;
 
-import static org.assertj.core.api.Assertions.assertThat;
-
 public class BedrockChatModelIT {
-
+    
+    private static final String CAT_IMAGE_URL = "https://upload.wikimedia.org/wikipedia/commons/e/e9/Felis_silvestris_silvestris_small_gradual_decrease_of_quality.png";
+    
+    @Test
+    @Disabled("To run this test, you must have provide your own access key, secret, region")
+    void testBedrockAnthropicV3SonnetChatModel() {
+        
+        BedrockAnthropicMessageChatModel bedrockChatModel = BedrockAnthropicMessageChatModel
+            .builder()
+            .temperature(0.50f)
+            .maxTokens(300)
+            .region(Region.US_EAST_1)
+            .model(BedrockAnthropicMessageChatModel.Types.AnthropicClaude3SonnetV1.getValue())
+            .maxRetries(1)
+            .build();
+        
+        assertThat(bedrockChatModel).isNotNull();
+        
+        Response<AiMessage> response = bedrockChatModel.generate(UserMessage.from("hi, how are you doing?"));
+        
+        assertThat(response).isNotNull();
+        assertThat(response.content().text()).isNotBlank();
+        assertThat(response.tokenUsage()).isNotNull();
+        assertThat(response.finishReason()).isIn(FinishReason.STOP, FinishReason.LENGTH);
```

Comment:
why is it either STOP or LENGTH?

Reply:
> why is it either STOP or LENGTH?

The generation varied in length for all the models so sometimes it would hit the token limit (300). I've changed this in the new PR so the prompt is a short yes/no answer.